error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java

```diff
@@ -138,4 +139,28 @@ ImmutableSet<Boolean> testStreamAllMatch() {
   boolean testStreamAllMatch2() {
     return Stream.of("foo").noneMatch(s -> !s.isBlank());
   }
+
+  ImmutableSet<Integer> testStreamMapToIntSum() {
+    Function<String, Integer> parseIntFunction = (String s) -> Integer.parseInt(s);
```

Comment:
Just because it's shorter:
```suggestion
    Function<String, Integer> parseIntFunction = Integer::parseInt;
```